deploy.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-multinetwork="$1"
-openshift_openstack_password="$2"
+multinetwork=$( grep multinetwork: environment.yaml | awk '{print $2}' )
```

Comment:
I wonder if it might be safer to use a YAML parser here. Try
`python -c "import yaml;d=yaml.load(open('environment.yaml'));print(d['parameter_defaults']['multinetwork'])"`

Reply:
additionally, deploy.sh should accept variations e.g. "false", "False", "FALSE" and convert them to the format we require. (e.g. fold the output to lower case)

Reply:
@andywis fancy checking the branch out and making those changes?